src/Pagination/PaginateDirective.php

```diff
@@ -7,6 +7,7 @@
 use GraphQL\Type\Definition\ResolveInfo;
 use Illuminate\Contracts\Pagination\Paginator;
 use Illuminate\Database\Eloquent\Builder as EloquentBuilder;
+use Illuminate\Database\Eloquent\Relations\Relation as EloquentRelation;
```

Comment:
We can leave this as `Relation` - in contrast to `Builder` there is no ambiguity.